pages/blog/[slug].js

```diff
@@ -50,14 +50,15 @@ const PostPage = (props) => {
 				</h1>
 				<p className="italic">{displayDate}</p>
 				<h3 className="mb-3"> By {data.blog.author}</h3>
+				<p className="italic lg:w-3/6 text-lg my-9">{data.blog.description}</p>
 				<img
 					src={data.blog.cover}
 					alt=""
 					width="650px"
 					className="my-12 max-h-[40rem] object-cover"
 				/>
 
-				<article className="prose prose-lg text-white prose-h3:text-white prose-a:text-slate-700 prose-strong:text-slate-700 prose-li:mb-1">
+				<article className="prose prose-lg text-white prose-h3:text-white prose-a:text-slate-700 prose-strong:text-slate-500 prose-h1:text-slate-500 prose-h2:text-slate-500 prose-em:italic prose-li:mb-1">
```

Comment:
I don't think this is a new change, but at large screen widths it is a little weird that the article body is like 1/2 width and left aligned. Can we center the body and make it wider at full-screen widths?

<img width="1470" alt="Screenshot 2024-03-02 at 7 45 48 PM" src="https://github.com/WXYC/website/assets/1800276/a974c3bd-7d53-4054-9de2-a5416d324f83">